### ISSUE TITLE
Document pilot telemetry collection

### ISSUE THREAD

**Issue by andraxylia** [istio/istio.github.io#879]

**Comment by nmittler** [comment]
@andraxylia what is this? To this point, it doesn't exist and it may never.

**Comment by nmittler** [comment]
@andraxylia ah ok, I think we can call this a duplicate of https://github.com/istio/istio/issues/2175 and just identify those subtasks that require documentation on the website.

**Comment by andraxylia** [comment]
This one is just about adding some documentation on istio.io website, about what metrics are exposed or general troubleshooting. If there is nothing relevant, please close it.

**Comment by nmittler** [comment]
I plan on covering everything required for completion of the issue in the subtasks of the epic.  If documentation is required, it should be there.  Please comment on those issues if you feel something is missing.

**Comment by nmittler** [comment]
@andraxylia also, I don't seem to have permission to close this issue :)

**Comment by andraxylia** [comment]
It's not a duplicate, but a child issue, 2175 is the Epic, it has 4 issues. When all 4 are closed, we can close the Epic and call it done.

**Comment by andraxylia** [comment]
@ldemailly can you please add @nmittler to Doc hackers so he can operate in this repo?

**Comment by ldemailly** [comment]
done

**Comment by nmittler** [comment]
@ldemailly thanks!

**Comment by ldemailly** [comment]
I think andra wanted this one to be closed only after the doc is live on site - is there a PR for the doc ?

**Comment by nmittler** [comment]
oh my apologies .... I had missed https://github.com/istio/istio.github.io/issues/879#issuecomment-359626536. Re-opening.

**Comment by nmittler** [comment]
@andraxylia regarding your comment https://github.com/istio/istio.github.io/issues/879#issuecomment-359626536, are you suggesting to leave this issue open? It seems that you're not opposed to closing this, but maybe I'm reading it wrong.

**Comment by andraxylia** [comment]
We close this issue only after the doc is available on istio.io.

**Comment by nmittler** [comment]
@andraxylia going back to my point on https://github.com/istio/istio.github.io/issues/879#issuecomment-359597091, we're tracking the same issue twice. https://github.com/istio/istio/issues/2175 was created a month ago and is mapped directly to the spreadsheet, so it's not clear why we have this issue. Documentation (if needed) should be added to the requirements for completion of the https://github.com/istio/istio/issues/2175 subtasks, not captured separately here.

**Comment by andraxylia** [comment]
Feel free to handle it as you see fit, as long as some documentation is added (when it makes sense).  Docs update are not always needed. In the past to tracked docs separately because they are in a separate repo, plus they were added/updated later in the cycle, and people preferred to close the coding issue sooner.

**Comment by nmittler** [comment]
@andraxylia yeah, makes sense. The epics that are mapped to the spreadsheet are a little different .. the requirements for closing them are that they're "feature complete", which includes docs (if necessary).

I'll close this issue and we can track docs in the other repo.

Thanks!
